packages/vue/src/slider/slider-thumb.tsx

```diff
@@ -10,17 +10,12 @@ export const SliderThumb: ComponentWithProps<SliderThumbProps> = defineComponent
   setup(_, { slots, attrs }) {
     const api = useSliderContext()
 
-    const hiddenInputProps = computed(() => ({
-      ...api.value.hiddenInputProps,
-      modelValue: api.value.value,
-    }))
```

Comment:
**question:** hiddenInputProps doesn't seem to include `api.value.value` so we might need to use a computed property after all if we want the input to have the current modelValue binding.


```diff
const hiddenInputProps = computed(() => ({
    ...api.value.hiddenInputProps,
-    modelValue: api.value.value,
+    value: api.value.value,
  }))
```